hardware/raspberrypi/booteeprom.md

```diff
@@ -6,104 +6,148 @@ The Raspberry Pi 4 has an SPI-attached EEPROM (4MBits/512KB), which contains cod
 If an error occurs during boot then an [error code](../../configuration/led_blink_warnings.md) will be displayed via the green LED. Newer versions of the bootloader will display a [diagnostic message](boot_diagnostics.md) which will be shown on both HDMI displays.
 
 ## Updating the bootloader
+### Pi 4 and Pi 400
+Raspberry Pi OS automatically updates the bootloader for critical bug fixes. The recommended methods for manually updating the bootloader or changing the boot modes are the Raspberry Pi Imager and raspi-config.
 
-## Compute Module 4
+### Compute Module 4
 Bootloader EEPROM updates on Compute Module 4 require [rpiboot](https://github.com/raspberrypi/usbboot) which is also used for flashing the EMMC. Please see the [Compute Module flashing guide](../computemodule/cm-emmc-flashing.md) for instructions.
 
-### Raspberry Pi Imager
-The easiest way to to update the bootloader to the latest version with default settings is to use the [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/) to install a boot recovery image onto a spare SD card.
+### Using Raspberry Pi Imager to update the bootloader (recommended)
+Raspberry Pi Imager provides a GUI for updating the bootloader and selecting the boot mode.
+
+1. Download [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/)
+2. Select a spare SD card. The contents will get overwritten!
+3. Launch `Raspberry Pi Imager`
+4. Select `Misc utility images` under `Operating System`
+5. Select `Bootloader`
+6. Select a boot-mode i.e. `SD` (recommended), `USB` or `Network`.
+7. Select `SD card` and then `Write`
+8. Boot the Raspberry Pi with the new image and wait for at least 10 seconds.
+9. The green activity LED will blink with a steady pattern and the HDMI display will be green on success.
+10. Power off the Raspberry Pi and remove the SD card.
+
+### Using raspi-config to update the bootloader from within Raspberry Pi OS
+To change the boot-mode or bootloader version from within Raspberry Pi OS run [raspi-config](../../configuration/raspi-config.md)
+
+1. [Update](../../raspbian/updating.md) Raspberry Pi OS to get the latest version of the `rpi-eeprom` package.
+2. Run `sudo raspi-config`
+3. Select `Advanced Options`
+4. Select `Bootloader Version`
+5. Select `Default` for factory default settings or `Latest` for the latest stable bootloader release.
+6. Reboot
 
-Select "Choose OS -> Misc utility images -> Raspberry Pi 4 EEPROM boot recovery"
+## Updating the EEPROM configuration
+The bootloader EEPROM image contains an embedded configuration file to define the boot behaviour (e.g. selecting between SD, network and USB boot).
+The `rpi-eeprom-config` tool may be used to modify embedded configuration file in an EEPROM image file.
```

Comment:
I would change `enbedded configuration filein an EEPROM image file` to `the configuration embedded in the EEPROM image`.

Reply:
reworded